google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/impl/GCStorageStream.scala

```diff
@@ -118,6 +118,7 @@ import scala.util.control.NonFatal
               .withQuery(Query(queryParams))
         )
       ).runWith(Sink.head)
+        .map(response => response.withEntity(response.entity.withoutSizeLimit))
```

Comment:
:+1:

Reply:
yes, saw that when listing a large bucket